client/my-sites/stats/stats-module/all-time-nav.jsx

```diff
@@ -0,0 +1,57 @@
+/**
+ * External dependencies
+ **/
+import React from 'react';
+import { connect } from 'react-redux';
+import { flowRight } from 'lodash';
+import { localize, moment } from 'i18n-calypso';
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import SelectDropdown from 'components/select-dropdown';
+import { recordGoogleEvent } from 'state/analytics/actions';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getSiteSlug } from 'state/sites/selectors';
+
+export const StatsModuleSummaryLinks = props => {
+	const { translate, path, siteSlug, query } = props;
+
+	const now = moment();
+	const quarter = moment().startOf( 'quarter' );
+	const daysSinceQuarterStart = now.diff( quarter, 'd' );
+	const options = [
+		{ value: '0', label: translate( 'Summary Views' ) },
+		{ value: '7', label: translate( 'Last 7 Days' ) },
+		{ value: '30', label: translate( 'Last 30 Days' ) },
+		{ value: `${ daysSinceQuarterStart }`, label: translate( 'This Quarter' ) },
```

Comment:
Alternate string suggestion: 
* `translate( 'the past quarter' )`&mdash; translations: **55**. 

[PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/stats/all-time-summary)